python/create_sandbox/dialog.py

```diff
@@ -90,7 +90,7 @@ def _process(self):
         """
         Creates a new configuration
         """
-        path = six.ensure_str(self.ui.path.text())
+        path = sgutils.ensure_str(self.ui.path.text())
```

Comment:
```suggestion
        path = str(self.ui.path.text())
```

So we don't need to require a new release of tk-core